standalone-node/docs/prepare_bootable_usb_windows.md

```diff
@@ -62,34 +94,66 @@ This document explains the procedure to create a bootable USB device for Standal
    ```shell
    usbipd bind --force --busid <busid for USB Drive>
    ```
-4. Start Ubuntu by running the command in PowerShell:
+4. Start Ubuntu:
 
-   ```shell
-   ubuntu2204.exe
-   ```
-   It will ask for the username and password you set previously. Upon successful login, the Ubuntu terminal will open.
+   a. (WSL) Run the command in PowerShell:
 
-5. Use PowerShell to attach the drive:
+      ```shell
+      ubuntu2204.exe
+      ```
+      It will ask for the username and password you set previously. Upon successful login,
+      the Ubuntu terminal will open.
 
-   ```shell
-   usbipd attach --wsl --busid <busid for USB Drive>
-   ```
+   b. (Hyper-V) Run Hyper-V Manager, right click your VM, and select *Connect...*.
+      Select *Start*.
+
+
+5. Attach the drive:
+
+   a. (WSL) Use PowerShell and run the command:
+
+      ```shell
+      usbipd attach --wsl --busid <busid for USB Drive>
+      ```
+
+   b. (Hyper-V) Start the Ubuntu terminal:
+
+
+      - Install Usbip tools:
```

Comment:
- Add troubleshooting tips for common errors (e.g., permissions, device not found).
- Provide example outputs for successful commands to help users verify progress.
- Link to official usbip documentation for advanced help.